nodejs_container/index.js

```diff
@@ -22,7 +23,26 @@ Container.prototype.call = function (id, zome, trait, fn, params) {
     return result;
 }
 
-module.exports = {
-    ConfigBuilder: new ConfigBuilder(),
-    Container: Container,
-};
+// Convenience function for making an object that can call into the container
+// in the context of a particular instance. This may be temporary.
+Container.prototype.makeCaller = function (agentId, dnaPath) {
```

Comment:
Why is this `makeCaller` and not `makeInstance` ?

Reply:
@lucksus it's just making an interface into the instance -- a lightweight object with `call` (just a combinator) and `agentId`. The actual instance is made during `new Container`.